DSCResources/xTimeZone/xTimeZone.psm1

```diff
@@ -13,7 +13,7 @@ function Get-TargetResource
     )
 
     #Get the current TimeZone
-    $CurrentTimeZone = Invoke-Expression "tzutil.exe /g"

```

Comment:
I updated return value to use the private function Get-TimeZone to make it easier to Mock in tests.